index.html

```diff
@@ -1,30 +1,26 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE HTML>
+<html>
+
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Colored Markers</title>
-    <link rel="stylesheet" href="stylesss.css">
+    <meta name="viewport" content="device-width" ,initial-scale 0.1>
+    <title>PAGES</title>
+    <link href="style.css" rel="stylesheet">
   </head>
 
   <body>
-    <script>const foo = 1
-      console.log(foo)
-        </script>
-    <h1>CSS Color Markers</h1>
-    <div class="container">
-      <div class="marker red">
-        <div class="cap"></div>
-        <div class="sleeve"></div>
-      </div>
-      <div class="marker green">
-        <div class="cap"></div>
-        <div class="sleeve"></div>
-      </div>
-      <div class="marker blue">
-        <div class="cap"></div>
-        <div class="sleeve"></div>
-      </div>
-    </div>
+    <h1>PAGES</h1>
+
+    <nav>
+      <h3>
+        <a href="http://127.0.0.1:5500/pages/cat/index.html">Cat</a>
+        <a href="http://127.0.0.1:5500/pages/coffee/index.html">Coffee</a>
+        <a href="http://127.0.0.1:5500/pages/markers/index.html">Markers</a>
```

Comment:
Use relative URLs. Seems like this was overwritten during last merge commit.

```suggestion
        <a href="./pages/cat/index.html">Cat</a>
        <a href="./pages/coffee/index.html">Coffee</a>
        <a href="./pages/markers/index.html">Markers</a>
```